offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/CreateOfferPresenter.groovy

```
@@ -50,4 +51,8 @@ class CreateOfferPresenter implements CreateOfferOutput {
        this.viewModel.failureNotifications.add(notification)
     }
 
+    @Override
+    void fetchedOffer(Offer fetchedOffer) {
+        this.createOfferViewModel.savedOffer = Optional.ofNullable(fetchedOffer)
```

```suggestion
        this.createOfferViewModel.savedOffer = Optional.of(fetchedOffer)
```

---

Fail fast principle.

---

I would also expect that the use case won't proxy a null object reference, that would be really bad.